### PR TITLE
Remove code coverage

### PR DESCRIPTION
**Pull request by ehalpern**
TBR

### CODE REVIEW

**Comment by ehalpern** [comment]
cc @rafael-atticlabs, @kalman

File: tools/run_pr_builder.sh [hunk 1]
```diff
@@ -17,18 +17,6 @@ node --version
 # go list is expensive, only do it once.
 GO_LIST="$(go list ./... | grep -v /vendor/ | grep -v /samples/js/)"
```
Comment (kalman) [inline tools/run_pr_builder.sh]: p.s. you can also remove the `node --version` above.

File: tools/run_pr_builder.sh [hunk 2]
```diff
@@ -17,18 +17,6 @@ node --version
...
-    if [ -f profile.out ]; then
-        cat profile.out >> coverage.txt
-        rm profile.out
-    fi
-done
+go test ${GO_LIST}
 
 python -m unittest discover -p "*_test.py" -s $GOPATH/src/github.com/attic-labs/noms/tools
-
-bash <(curl -s https://codecov.io/bash) -t ${COVERALLS_TOKEN}
```
Comment (kalman) [inline tools/run_pr_builder.sh]: not sure if you noticed, but you can remove the COVERALLS_TOKEN thing from jenkins after this

**Comment by ehalpern** [comment]
done

**Pull request merged**